src/csharp/Extensions/Microsoft.Spark.Extensions.Hyperspace/HyperspaceVersions.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Spark.Extensions.Hyperspace
+{
+    internal static class HyperspaceVersions
+    {
+        internal const string V0_0_1 = "0.0.1";
+        internal const string V0_0_2 = "0.0.2";
```

Comment:
I don't see this version being attributed anywhere, is there a reason we have included it here?

Reply:
I included it just for completeness.  @suhsteve Do you think we should include only the necessary versions?

Reply:
We can just add new versions as necessary, ie when we need to use it.

Reply:
Sounds good, removed `0.0.2`.